buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -54,7 +55,7 @@ fun Project.configureIntellijPlugin() {
       val prospectiveVersionSection = changelog.get(changelog.version.get())
       val latestVersionSection = changelog.getLatest()
 
-      if (prospectiveVersionSection.version != latestVersionSection.version) {
+      if (prospectiveVersionSection.version != latestVersionSection.version) { // here
```

Comment:
the problem is that the latest ignores unreleased. what should we do in that case @PawelLipski? it can be resolved by removing `unreleasedTerm.set("v$PROSPECTIVE_RELEASE_VERSION")` in line 46

Reply:
Okay... is there gonna be any change to the overall behavior (esp. of the two custom `verifyChangeLog...` tasks) if we do so? 🤔

Reply:
actually this if statement will be redundant -latest will equal to prospective

Reply:
Even better!

Reply:
we can go even further...